src/components/pages/Flagging/FlagFeed.tsx

```diff
@@ -0,0 +1,210 @@
+/**
+ * The Feed is a great place for members to catch up on what's happened recently!
+ * Shows all of the most recent raha gives and verified join videos.
+ * We should add ability to see only transactions of people you trust.
+ */
+import * as React from "react";
+import { View, FlatList, StyleSheet, ViewStyle, TextStyle } from "react-native";
+import { connect, MapStateToProps } from "react-redux";
+import { formatRelative } from "date-fns";
+import { NavigationScreenProps } from "react-navigation";
+import { List } from "immutable";
+
+import { FlagMemberOperation } from "@raha/api-shared/dist/models/Operation";
+
+import { RahaState } from "../../../store";
+import { colors } from "../../../helpers/colors";
+import { Text, Button } from "../../shared/elements";
+import { MemberName } from "../../shared/MemberName";
+import { Member } from "../../../store/reducers/members";
+import { getMemberById } from "../../../store/selectors/members";
+import { fontSizes } from "../../../helpers/fonts";
+import { MemberThumbnail } from "../../shared/MemberThumbnail";
+import { RouteName } from "../../shared/Navigation";
+import { generateRandomIdentifier } from "../../../helpers/identifiers";
+import { styles as sharedStyles } from "./styles";
+
+interface FlagData {
+  flaggingMember: Member;
```

Comment:
Is this the member who is flagging or who is being flagged? Can we say memberToFlag or something to be clear?

Reply:
renamed to `flaggedByMember`